docs/cqd_reference/src/asciidoc/_chapters/operational_controls.adoc

```diff
@@ -53,12 +53,89 @@ then a large number of queries may be impacted. Therefore, you need to assess if
 |===
 
 <<<
+[[explain-description-column-size]]
+== EXPLAIN_DESCRIPTION_COLUMN_SIZE
+
+[cols="25%h,75%"]
+|===
+| *Description*               | Specifies maximum length of DESCRIPTION column for EXPLAIN virtual tables.
+| *Values*                    |
+Positive integer > 10,000 +
+ +
+Default: -1, indicating a maximum size of 10,000.
+| *Usage*                     | Query-plan information for a SQL DML statement is stored temporarily in the Explain virtual table. 
+For large queries or queries with complex predicates, the default size of 10 KB may be insufficient to describe certain nodes in the query plan. 
+Specifying a larger value for this CQD allows more bytes to be stored in the description column. +
+ +
+Change this setting if you see explain plan being undesirably truncated only.
+| *Production Usage*          | Yes.
+| *Impact*                    | The explain plan truncation is reduced or removed.
+| *Level*                     | System.
+| *Conflicts/Synergies*       | Not applicable.
+| *Real Problem Addressed*    | Not applicable.
+| *Introduced In Release*     | Trafodion 1.3.0.
+| *Deprecated In Release*     | Not applicable.
+|===
+
+<<<
+[[hbase-region-server-max-heap-size]]
+== HBASE_REGION_SERVER_MAX_HEAP_SIZE
+
+[cols="25%h,75%"]
+|===
+| *Description*               | Enables Trafodion to use HBase block cache in an optimal manner.  Defines the maximum Java
+heap size (-Xmx option) the HBase Region Server are assigned, in MB units.
+| *Values*                    |
+Positive integer. +
+ +
+Default: *'1024'*.
+| *Usage*                     | If the HBase Region servers are configured with a maximum Java heap size different than 1 GB, then
+set this attribute so that Trafodion is aware of the actual maximum heap size of the Region Servers.
+| *Production Usage*          | Yes.
+| *Impact*                    | When set correctly, this CQD ensures that HBase block cache are be used optimally. Small scans
+are cached and larger scans are not cached to avoid cache trashing.
+| *Level*                     | System.
+| *Conflicts/Synergies*       | Not applicable.
+| *Real Problem Addressed*    | Not applicable.
+| *Introduced In Release*     | Trafodion 1.3.0.
+| *Deprecated In Release*     | Not applicable.
+|===
+
+<<<
+[[hive-metadata-refresh-interval]]
+== HIVE_METADATA_REFRESH_INTERVAL
+
+[cols="25%h,75%"]
+|===
+| *Description*               | Controls the metadata cache for Hive tables.
+| *Values*                    | 
+*-1*: Never invalidate any cache entries. +
+*0*: Always check the latest meta-data from Hive. +
+*> 0*: The cached Hive meta-data is valid only for <value> seconds. +
+ +
+Default: *'0'*.
+| *Usage*                     | Use of value -1 when the Hive tables are read-only and no updates are possible to avoid reading the
```

Comment:
Suggested wordsmith: "Use a value of -1 when the Hive tables are read-only. This allows Trafodion to avoid repeatedly reading the metadata when the table is referenced in multiple queries.